TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/DemoAPI.kt

```diff
@@ -0,0 +1,48 @@
+package org.firstinspires.ftc.teamcode.api
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode
+import com.qualcomm.robotcore.hardware.DcMotor
+import org.firstinspires.ftc.teamcode.core.API
+
+/**
+ * This is a demo API used for controlling a robot with four mecanum wheels.
+ */
+object DemoAPI : API() {
+    // The wheel motors
+    // `lateinit` means the variable does not have to be initialized in the constructor
+    // See: https://github.com/BotsBurgh/BOTSBURGH-FTC-2023-24/pull/7#discussion_r1342154226
+    private lateinit var fl: DcMotor
+    private lateinit var fr: DcMotor
+    private lateinit var bl: DcMotor
+    private lateinit var br: DcMotor
+
+    // `init()` is run once at the beginning of the program.
+    // It initializes things and sets up configuration
```

Comment:
```suggestion
    // `init()` is called by the opmode in order to use this API.
    // Override it if you need do things at startup, such as accessing hardware.
```

I would rephrase this to show that opmodes have to manually call `MyAPI.init()`. I feel like your original wording makes it seem like this is done automatically.

I wrote a suggestion that you can use or rephrase if you want!